package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.8",
+    "modal": "^1.2.0",
```

Comment:
Did you mean to install [this](https://www.npmjs.com/package/modal)?

You can uninstall it by running `npm uninstall modal` in the terminal

Reply:
we uninstalled it. But as it still works we are not sure if we got your point right ;-)

Reply:
I don't think you were using it in your project. So the fact that you uninstalled it and everything still works is a good sign.